File: app/src/main/java/com/wiktor/weater1/newWeaher/cityList/mvp/CityListModel.java
package com.wiktor.weater1.newWeaher.cityList.mvp;

import com.wiktor.weater1.newWeaher.cityList.model.CityModelKt;

import java.util.ArrayList;
import java.util.List;

public class CityListModel implements CityListContract.Model {

    ArrayList<CityModelKt> listCities;

    @Override
    public List<CityModelKt> getListOfCities() {
        return listCities;
    }

    public CityListModel() {
        listCities = new ArrayList<>();
        listCities.add(new CityModelKt("Москва", "Moscow"));
        listCities.add(new CityModelKt("Париж", "Paris"));
        listCities.add(new CityModelKt("Лондон", "London"));
        listCities.add(new CityModelKt("Нью-Йорк", "new-york"));
        listCities.add(new CityModelKt("Милан", "milan"));
        listCities.add(new CityModelKt("Рим", "rome"));
        listCities.add(new CityModelKt("Каир", "Cairo"));
        listCities.add(new CityModelKt("Минск", "Minsk"));
        listCities.add(new CityModelKt("Стокгольм", "Stockholm"));
        listCities.add(new CityModelKt("Рейкьявик", "Reykjavik"));
        listCities.add(new CityModelKt("Крайстчерч", "Christchurch"));
        listCities.add(new CityModelKt("Кейптаун", "Cape Town"));
        listCities.add(new CityModelKt("Гавана", "Habana"));
        listCities.add(new CityModelKt("Рио-де-Жанейро", "Rio de Janeiro"));
        // запрос работает с русскими названияи
        listCities.add(new CityModelKt("Архангельск", "Архангельск"));
    }
}

File: app/src/main/java/com/wiktor/weater1/newWeaher/cityList/mvp/CityListModel.kt
package com.wiktor.weater1.newWeaher.cityList.mvp

import com.wiktor.weater1.newWeaher.cityList.model.CityModel
import java.util.*


class CityListModel : Model {

    internal var listCities: ArrayList<CityModel> = ArrayList()

    override fun getListOfCities(): List<CityModel> {
        return listCities
    }

    init {
        listCities.add(CityModel("Москва", "Moscow"))
        listCities.add(CityModel("Париж", "Paris"))
        listCities.add(CityModel("Лондон", "London"))
        listCities.add(CityModel("Нью-Йорк", "new-york"))
        listCities.add(CityModel("Милан", "milan"))
        listCities.add(CityModel("Рим", "rome"))
        listCities.add(CityModel("Каир", "Cairo"))
        listCities.add(CityModel("Минск", "Minsk"))
        listCities.add(CityModel("Стокгольм", "Stockholm"))
        listCities.add(CityModel("Рейкьявик", "Reykjavik"))
        listCities.add(CityModel("Крайстчерч", "Christchurch"))
        listCities.add(CityModel("Кейптаун", "Cape Town"))
        listCities.add(CityModel("Гавана", "Habana"))
        listCities.add(CityModel("Рио-де-Жанейро", "Rio de Janeiro"))
        // запрос работает с русскими названияи
        listCities.add(CityModel("Архангельск", "Архангельск"))
    }
}
